crates/net/network/src/manager.rs

```diff
@@ -174,13 +175,16 @@ where
         // need to retrieve the addr here since provided port could be `0`
         let local_peer_id = discovery.local_id();
 
+        let bandwidth_meter: BandwidthMeter = BandwidthMeter::new();
```

Comment:
@mattsse Should I add the `BandwidthMeter` to the `NetworkManager` struct as well?

As of yet it doesn't need to be referenced off of the `NetworkManager` anywhere but it feels appropriate given the `BandwidthMeter` used by the `NetworkInner` and the `SessionManager` is created in `NetworkManager::new`, I could see this being useful

Reply:
since we have this in `NetworkInner` we can expose this

```rust

impl NetworkHandle {
  pub fn bandwith_meter() {
       &self.inner.bandwith_meter
  } 
}

impl NetworkManager {
  pub fn bandwith_meter() {
       self.handle.bandwith_meter()
  } 
}

```

so we don't need to add it to the Manager